examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -28,6 +28,89 @@ Path _triangle(double size, Offset thumbCenter, {bool invert = false}) {
   return thumbPath;
 }
 
+Path _leftTriangle(double size, Offset thumbCenter, {bool invert = false}) {
+  final Path thumbPath = Path();
+  final double halfSize = size / 2.0;
+  final double sign = invert ? -1.0 : 1.0;
+  thumbPath.moveTo(thumbCenter.dx - halfSize, thumbCenter.dy);
```

Comment:
The `- halfSize` in this line (and the `+ halfSize` in the lines below) seem to be the only difference between `_leftTriangle` and `_rightTriangle`, any chance you can combine the two functions?

Reply:
Yeah I agree, consolidate these 2 functions with a parameter to specify direction

Reply:
Refactored the triangle drawing to be consistent with the existing "invert" option in the up and down triangles